src/simpla-img.html

```diff
@@ -100,35 +108,36 @@
        * @return {undefined}
        */
       _styleEditable(editable) {
-        this._toggleStyles(CONDITIONAL_STYLES.editable, editable);
+        this._toggleStyles(IMG_STYLES.editable, editable);
       },
 
       /**
-       * Set inline styles for editing state
-       * @param  {Boolean} editing Value of the editing prop
+       * Set inline styles for active state
+       * @param  {Boolean} active Value of the active prop
        * @return {undefined}
        */
-      _styleEditing(editing) {
-        this._toggleStyles(CONDITIONAL_STYLES.editing, editing);
+      _styleEditing(active) {
+        this._toggleStyles(IMG_STYLES.active, active);
       },
 
       /**
        * Fake a basic placeholder with inline styles
-       * @param  {Boolean} editable Value of the editable prop
-       * @param  {Boolean} editing Value of the editable prop
-       * @param  {String} src       Value of the src prop
+       * @param  {Boolean} placeholder String to apply to background
```

Comment:
Random aside - I've started being more descriptive in my JSDoc comments ('Value of the editable prop' isn't very helpful), going to try and follow this pattern of what the arg is actually used for in the function/by the user. Otherwise they end up like this https://twitter.com/randal_olson/status/843831671648669696